sb.web.app/src/components/Home/ListElement.vue

```diff
@@ -5,8 +5,8 @@
           <img v-if="elementInfo.coverImg.length === 0" class="regular" :src="require(`../../assets/list_images/event.png`)" style="object-fit:contain;max-width:240px;max-height:240px;">
           <img v-else class="regular" :src="getArticleImgURL()" style="object-fit:contain;max-width:240px;max-height:240px;">
           <h4 class="element-title"> {{ elementInfo.title }} </h4>
-          <p v-if="elementInfo.status === 'selling'" class="element-desc"> {{ $t('offer') }} </p>
-          <p v-if="elementInfo.status === 'buying'" class="element-desc">  {{ $t('need') }} </p>
+          <p v-if="elementInfo.status === 'selling' || elementInfo.status === 'offer' " class="element-desc"> {{ $t('offer') }} </p>
+          <p v-if="elementInfo.status === 'buying' || elementInfo.status === 'want'" class="element-desc">  {{ $t('need') }} </p>
```

Comment:
Can you add that issue? Or failing that we need to do this via a migration in the database?

Reply:
let's create that issue because writing migration scripts might be tedious and on an unstructured database like Mongo might not work as intended so I will create an issue for it